docs(gh_actions): fixes #1573 (#1574)
psf/black: docs/integrations/github_actions.md

    runs-on: ubuntu-latest
    steps:
      - uses: actions/checkout@v2

>       # you can change the tag @stable with any of the available here: https://github.com/psf/black/tags

Reviewer: ```suggestion
      # you can change "stable" to a specific release: https://github.com/psf/black/tags
```
